.dockerignore

```diff
@@ -0,0 +1,5 @@
+Dockerfile
+.git
+node_modules
+npm-debug.log*
+webhooks.json
```

Comment:
* [ ] Add `lib` folder to .dockerignore, which is generated when you build the project locally